### PR TITLE
Localized strings in call and chat composites

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

Localized strings in call composite

# Why

https://skype.visualstudio.com/SPOOL/_workitems/edit/2555551

# How Tested

Checked that calling sample and composite in storybook is working.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by anjulgarg** [comment]
Are we planning to add other languages for these strings?

File: packages/react-composites/src/composites/CallComposite/StartCallButton.tsx [hunk 1]
```diff
@@ -5,16 +5,18 @@ import { PrimaryButton } from '@fluentui/react';
...
   isDisabled: boolean;
 }
 
-const buttonText = 'Start call';
-
 export const StartCallButton = (props: StartCallButtonProps): JSX.Element => {
   const { isDisabled, onClickHandler } = props;
+  const locale = useLocale();
+
+  const buttonText = locale.strings.call.startCallButtonText;
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/CallComposite/StartCallButton.tsx:19]: nit: Simply inline in line 29. It's used exactly once.

**Review by prprabhu-ms — APPROVED** [review]

File: common/scripts/ensure-consistent-versions.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
```
Comment (anjulgarg) [inline common/scripts/ensure-consistent-versions.js:1]: Why is this script here?
I thought it was part of your PR @prprabhu-ms  ?
cc @mgamis-msft

**Comment by mgamis-msft** [comment]
> Are we planning to add other languages for these strings?

Yes, but that is handled by a separate CI after this has been merged to main.

File: common/scripts/ensure-consistent-versions.js (hunk 2 above)
Comment (mgamis-msft) [reply common/scripts/ensure-consistent-versions.js]: Not sure how this happened after I merged main to my branch but I have now pruned these other changes from this PR.

**Review by alcail — APPROVED** [review]

**Pull request merged**